user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md

```diff
@@ -251,7 +252,12 @@ const handler = getHandler(myContract)(async event => {
 ```
 
 :::info
-By default, the `getHandler` feature will validate both the input and the output of your lambda. If you wish to disable one of those, you can use the optional second argument in the `getHandler` feature:
+Regarding the `ajv`option, we advise you to use a singleton instance of ajv that you define in a separate file. This way, you can use the same instance for all your contracts and middlewares.
+:::
```

Comment:
This line is useless

```suggestion
```

Reply:
Je viens de tester, et ça casse tout si je l'enlève